Product/Production/Services/DocumentQueryCore/src/main/java/gov/hhs/fha/nhinc/docquery/audit/transform/DocQueryAuditTransforms.java

```diff
@@ -251,9 +253,10 @@ private String getPatientIdFromRequest(AdhocQueryRequest request) {
             && request.getAdhocQuery().getSlot() != null
             && !request.getAdhocQuery().getSlot().isEmpty()) {
             for (SlotType1 slot : request.getAdhocQuery().getSlot()) {
-                if (slot.getName().equals(DocQueryAuditTransformsConstants.XDS_DOCUMENT_ENTRY_PATIENT_ID)) {
-                    String value = slot.getValueList().getValue().toString();
-                    return value.replaceAll("\\[|\\]", "");
+                if (slot != null && slot.getName().equals(DocQueryAuditTransformsConstants.XDS_DOCUMENT_ENTRY_PATIENT_ID)
```

Comment:
Do we need to check value of slot.getName() for null? Or we are sure that value will always be there?